src/cmd/builtin/uptime.c

```diff
@@ -229,29 +229,52 @@ static void append_count_if (char *buf,
  */
```

Comment:
commit message typo

```Problem: if the broker gets stuck in rc1, cleanup, or rc3,
it May not be evident what is going on without examining logs.
```

lower case `M` in `May`.